### ISSUE TITLE
can't install anymore

### ISSUE THREAD

**Issue by sragneau** [openvenues/node-postal#14]
Hi,

Since i'd like to reinstall in the aim to test the 1.0 version, i have some error with node-gyp build.
```
> node-postal@1.0.0 install /home/sragneau/node-postal
> (node-gyp rebuild) || (exit 0)

make : on entre dans le répertoire « /home/sragneau/node-postal/build »
  CXX(target) Release/obj.target/expand/src/expand.o
../src/expand.cc: In function ‘Nan::NAN_METHOD_RETURN_TYPE ExpandAddress(Nan::NAN_METHOD_ARGS_TYPE)’:
../src/expand.cc:26:5: error: ‘libpostal_normalize_options_t’ was not declared in this scope
     libpostal_normalize_options_t options = libpostal_get_default_options();
     ^
../src/expand.cc:55:64: error: ‘LIBPOSTAL_MAX_LANGUAGE_LEN’ was not declared in this scope
                             if (lang != NULL && strlen(lang) < LIBPOSTAL_MAX_LANGUAGE_LEN) {
                                                                ^
../src/expand.cc:63:25: error: ‘options’ was not declared in this scope
                         options.address_components = (uint16_t) Nan::To<uint32_t>(value).FromJust();
                         ^
../src/expand.cc:66:21: error: ‘options’ was not declared in this scope
                     options.latin_ascii = Nan::To<bool>(value).FromJust();
                     ^
../src/expand.cc:68:21: error: ‘options’ was not declared in this scope
                     options.transliterate = Nan::To<bool>(value).FromJust();
                     ^
../src/expand.cc:70:21: error: ‘options’ was not declared in this scope
                     options.strip_accents = Nan::To<bool>(value).FromJust();
                     ^
```

It is cause by my machine or by the new version ?

Thanks you

**Comment by thatdatabaseguy** [comment]
Did you upgrade the C library as well? If not, pull latest from https://github.com/openvenues/libpostal, and run:

```shell
./bootstrap.sh
./configure --datadir=[...some dir with a few GB of space...]
make
sudo make install

# On Linux it's probably a good idea to run
sudo ldconfig
```

Then try building the Node binding.

**Comment by sragneau** [comment]
Thx it was just that... :)

**Comment by thatdatabaseguy** [comment]
Glad it worked!

**Comment by jlbribeiro** [comment]
@thatdatabaseguy Actually I'm having the exact same issue (same error message as above) on a fresh install of both the C library and the `node-postal` library. Just re-ran the commands you posted above and everything seemed to run ok.
The only thing I can think of is that the C library installation steps are being performed by a different user that the one `npm install`ing/`npm rebuild`ing (because of `sudo`), but I'm not sure how that would make a difference.

**Comment by thatdatabaseguy** [comment]
Hey @jlbribeiro, which OS? On Linux you might need to run ```sudo ldconfig``` to make sure it's linking to the correct C library.

**Comment by jlbribeiro** [comment]
@thatdatabaseguy
*OS:* Ubuntu 16.04.2 LTS.
I already did a `sudo ldconfig`, and an `npm rebuild node-postal` afterwards, but the error still occurs. Is there some other step I may be missing?

**Comment by thatdatabaseguy** [comment]
@jlbribeiro, wait, why rebuild instead of install? Did you pull the latest node-postal as well?

**Comment by jlbribeiro** [comment]
@thatdatabaseguy I had already tried reinstalling, but... you're right, I'm not :( I'm installing a [Pelias](https://github.com/pelias/pelias) instance and unfortunately its dependencies seem to be installing `node-postal` `0.3.0` (I just noticed `node-postal`'s version just got bumped 10 days ago, hence the _freshness_ of the bug). Either I downgrade the C library or I'll have to report this upstream. Thank you for your quick support!
